editor/src/components/editor/store/editor-state.ts

```diff
@@ -3582,6 +3582,9 @@ export function modifyUnderlyingTarget(
     underlyingFilePath: string,
   ) => ParseSuccess = defaultModifyParseSuccess,
 ): EditorState {
+  if (target == null) {
```

Comment:
Surely the function signature should be updated to remove the `| null` then?!

Reply:
I somewhat assumed doing that would be the proverbial pulling on a thread, turns out nothing passes null for that parameter. Fixed in 380a604af.

Reply:
Beautiful!